app/controllers/users_controller.rb

```diff
@@ -49,7 +49,7 @@ class UserController < ApplicationController
   get '/logout' do
     if logged_in?
       session.clear
-      flash[:message] = 'You have logged out successfully!'
+      flash[:error] = 'You have logged out successfully!'
```

Comment:
Do you really need this?

Reply:
Yea not really but just a convention it should be message not error !!! Thanks